.travis.yml

```diff
@@ -0,0 +1,32 @@
+language: node_js
+sudo: required
```

Comment:
why is this?

Reply:
Good catch.  This is no longer necessary with the updated encryption/upload scheme.